tools/ci_build/build.py

```diff
@@ -1163,28 +1163,27 @@ def run_training_python_frontend_e2e_tests(cwd):
     # frontend tests are to be added here:
     log.info("Running python frontend e2e tests.")
 
+    run_subprocess(
```

Comment:
i think build.py has a lot of stuff in it already. can this script be invoked from a build definition instead?
or maybe we should rename build.py :)

Reply:
orttraining-linux-gpu-e2e-test-ci-pipeline.yml is a good example for doing e2e tests in a build definition. I see many benefits of it. I will do as you suggested next when merging cpp e2e and python e2e pipelines into one pipeline.